packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-import * as api from '@opentelemetry/api';
+import * as api from "@opentelemetry/api";
```

Comment:
It looks like you changed quite a few quotes from ' to " is there a reason for this?

Reply:
Ignore this i had a commit filter on